spring-boot/admin/src/main/resources/messages.properties

```
@@ -189,9 +201,23 @@ admin.help.getting.started.gateway.url.heading=Define the public URL of your gat
 admin.help.getting.started.gateway.url.text= Please insert the public URL where API user clients can reach your installed CoatRack gateway. CoatRack will also use this URL to notify the gateway about configuration updates.
 admin.help.getting.started.gateway.url.image.path=/images/coatrack-getting-started-gateway-url.png
 admin.help.getting.started.gateway.url.image.alt=CoatRack overview and first steps - Gateway URL
+
 admin.help.getting.started.api.key.heading=API key to be used in service calls via CoatRack
 admin.help.getting.started.api.key.text= Service users need to pass this key in each call to a service via CoatRack. CoatRack will check this key in order to authenticate/authorize the access.
 admin.help.getting.started.api.key.image.path=/images/coatrack-getting-started-api-key.png
 admin.help.getting.started.api.key.image.alt=CoatRack overview and first steps - API key
+
+admin.help.gateway.health.monitor.heading=Gateway Health Monitor Introduction
+admin.help.gateway.health.monitor.example.image=/images/coatrack-gateway-health-monitor-example.png
+admin.help.gateway.health.monitor.example.image.alt=An example of a CoatRack Gateway Health Monitor.
+admin.help.gateway.health.monitor.text.summary=<b>Summary</b><br> If at least one gateway is in warning state (orange) or critical state (red) the summary at the top will display that there is a problem with your gateway(s). You might want to check it out.
+admin.help.gateway.health.monitor.text.ok=<b>Green status (OK)</b><br> The gateway successfully contacted the CoatRack admin server within the last 5 minutes.
+admin.help.gateway.health.monitor.text.warning=<b>Orange status (warning)</b><br> The gateway did not connect to the CoatRack admin server for longer than 5 minutes.
+admin.help.gateway.health.monitor.text.critical=<b>Red status (critical)</b><br> The gateway did not connect to the CoatRack admin server for longer than 60 minutes.
+admin.help.gateway.health.monitor.text.not.connected=<b>Dark Grey status (not connected)</b><br> Since the creation of the gateway, no connection/call was done to CoatRack admin server.
+admin.help.gateway.health.monitor.text.ignore=<b>Light Grey status (ignore)</b><br> The status of the gateway is ignored by manually deactivate the monitoring switch.
+admin.help.gateway.health.monitor.text.disclaimer=<b>Disclaimer</b><br> On the example provided here, the considered values for the warning threshold and critical threshold are 5 and 60 minutes.
```

The word `disclaimer` is misleading here. Maybe `note` would be better. E.g.:
```suggestion
admin.help.gateway.health.monitor.text.note=<b>Please note</b><br> In this example, the threshold times for warning and critical state are 5 and 60 minutes, respectively.
```